config/locales/en.yml

```diff
@@ -23,6 +23,7 @@ en:
     paranoid_verify:
       code_required: 'Please enter the code our support team provided'
     paranoid_verification_code:
+      updated: Verification code accepted
```

Comment:
This will probably raise an error in non-english localizations. Contributions are welcome.